compiler/modules/front.syntax/token.bal

```diff
@@ -330,6 +330,10 @@ class Tokenizer {
         self.mode = s.mode;
         self.curTok = s.curTok;
 
+        if self.lineIndex == 0 {
```

Comment:
In this case I would set fragCodes and fragments to [], so it's the same as at initialization.